modules/nodes-pods-autoscaling-custom-gather.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-gather_{context}"]
+= Gathering data about your cluster
+
+When opening a support case, it is helpful to provide debugging information about your cluster to Red Hat Support.
+
+It is recommended to provide:
+
+* Data gathered using the `oc adm must-gather` command
```

Comment:
I suggest starting with the specific CMA's  command (see below), as `oc adm must-gather`  will not collect CMA resources.

```
oc adm must-gather --image="$(oc get packagemanifests openshift-custom-metrics-autoscaler-operator \
-n openshift-marketplace \
-o jsonpath='{.status.channels[?(@.name=="stable")].currentCSVDesc. annotations.containerImage}')"
```

Reply:
@jrvaldes You can use [the `oc adm must-gather` tool with the CMA image](https://github.com/openshift/openshift-docs/pull/57331/files#diff-ca581f5080fd96362d3d11f8c367f90056995b5ceb40ba8c79ec0708d003877cR156), right?

Reply:
Correct, that is the [proposed](https://github.com/openshift/custom-metrics-autoscaler-operator/pull/21/files#diff-f22cb59a526d4b8f524cafa3abb53f4ced4afc9b32ef3b3a420ed87e4b870d2eR1) approach.

Reply:
@jrvaldes Just want to confirm, other for other operators, we [list the image name in the must-gather docs](https://docs.openshift.com/container-platform/4.12/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data).  Do we have an image name that we can list, rather than running the command to fetch the image?

Reply:
xref to [thread](https://redhat-internal.slack.com/archives/GK6BJJ1J5/p1679671860456019) in Slack